backgroundpage.js

```diff
@@ -239,7 +239,7 @@ var arr = [],
 
 /*Creating a new tab if notification is clicked*/
 chrome.alarms.create("routine", {
-    delayInMinutes: .1
+    delayInMinutes: 1
```

Comment:
It keeps telling me the minimum is 1, and that in the packed extension it will be set to 1.